q2_quality_control/_blast.py

```diff
@@ -11,9 +11,9 @@
 import pandas as pd
 
 
-def _blast_seqs(feature_sequences, reference_sequences, evalue,
-                perc_identity, threads, perc_query_aligned,
-                method='blast'):
+def _search_seqs(feature_sequences, reference_sequences, evalue,
+                 perc_identity, threads, perc_query_aligned,
+                 method='blast'):
```

Comment:
Unless there is a good reason to keep it, I suggest removing the default of `blast` from the private method.

Reply:
👍